cms/templates/js/video-transcript-upload-status.underscore

```diff
@@ -0,0 +1,5 @@
+<span class="fa <%- iconClasses %>" aria-hidden="true"></span>
+<span class='transcript-detail-status'><%- shortMessage %></span>
+<button class='button-link more-details-action <%- hiddenClass %>' data-error-message='<%- errorMessage %>'>
+    <%- gettext('Read More') %>
```

Comment:
User facing string

Reply:
I'm not sure where this appears, and that might affect my suggestion, but I think it should be "Read more."